document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/mapping/model/offence/Offence.java

```diff
@@ -10,18 +10,12 @@
     @JsonProperty("filing_period_type")
     private FilingPeriodType filingPeriodType;
 
-    @JsonProperty("status")
-    private OffenceStatus status;
-
     @JsonProperty("filing_period_ends_on")
```

Comment:
The casing of the JSON property annotations is inconsistent

Reply:
Changed case to be consistent with the other models in the service. The JSON properties have used the same case as the Go template standards as we have agreed prior to me starting work on the story.